QuizBowlDiscordScoreTracker/Web/wwwroot/js/main.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
+var nameDisplay = document.getElementById("name");
+var sound1 = document.getElementById("sound1");
+
+var shouldEmitSound = true;
+
+function removeBarrier() {
+    document.getElementById("interactBarrier").style.display = "none";
+}
+
+connection.on("PlayerBuzz", function (name) {
+    document.body.style.backgroundColor = "red";
+    nameDisplay.textContent = name;
+
+    if (shouldEmitSound) {
+        sound1.play();
+    }
+
+    shouldEmitSound = false;
+});
+
+connection.on("Clear", function () {
+    document.body.style.backgroundColor = "green";
+    nameDisplay.textContent = "";
+    shouldEmitSound = true;
+});
+
+connection.start().then(function () {
+    connection.invoke("AddToChannel", window.location.search.substring(1));
```

Comment:
Should we only invoke if window.location.search.length > 1? This won't throw, but there's no need to add stuff for a channel that can't exist ("").

Reply:
fixed in ca6ff91